dns_check/datadog_checks/dns_check/dns_check.py

```diff
@@ -105,7 +109,7 @@ def _check_answer(self, answer):
 
         assert len(self.resolves_as_ips) == number_of_results
         result_ips = []
-        for rip in answer.rrset.items:
+        for rip in answer.rrset.items.keys() if PY3 else answer.rrset.items:
```

Comment:
Wouldn't this be equivalent regardless of whether `items` is a list or a dict? dicts iterate on keys by default.

```suggestion
        for rip in answer.rrset.items:
```